caluma/extensions/common.py

```diff
@@ -1,8 +1,26 @@
+from django.core.cache import cache
+
+from caluma.extensions.settings import settings
+
 from .api_client import APIClient
 
 
 def get_api_user_attributes(token, idp_id):
-    client = APIClient(token=token)
-    result = client.request(f"/identities?filter%5BidpIds%5D={idp_id}")
+    client = APIClient()
+    result = client.get(f"/identities?filter%5BidpIds%5D={idp_id}", token)
```

Comment:
I preferred the old calling convention here. Why the change?

Reply:
The reason for this is, usually we use the token from the request to caluma. Just when creating the initial CaseAccess, we have to fetch one with the confidential client.
I've decided to not put too much magic in the client to handle this. So you can make requests with whatever token you already have or one you just fetched with the client.